src/reversion/management/commands/deleterevisions.py

```diff
@@ -71,7 +78,14 @@ def handle(self, *app_labels, **options):
         days = options["days"]
         keep = options["keep"]
         force = options["force"]
-        confirmation = options["confirmation"]
+        interactive = options.get("interactive")
+        if not options.get("confirmation"):
+            interactive = False
+        warnings.warn(
```

Comment:
oops! I de-dented it to make it warn all the time and forgot to undo before committing.